engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsNode.java

```diff
@@ -176,14 +176,22 @@ private static boolean findConversionImpl(
           UnresolvedConversion.build(selfScope).resolveFor(ctx, comparableType, thatType);
```

Comment:
[Question not related to the current PR] Why is this conversion built with `selfScope` and not `thatScope`? We are looking for a conversion from `thatType` to `Comparable` type in `selfScope`, but shouldn't we be looking for this conversion in a different scope? Is is possible that this is some leftover? @JaroslavTulach